engine/src/main/java/org/terasology/entitySystem/entity/internal/PojoEntityManager.java

```diff
@@ -250,31 +213,17 @@ public EntityRef getEntity(long id) {
     }
 
     @Override
-    public EntityRef create(Prefab prefab, Vector3f position) {
-        List<Component> components = Lists.newArrayList();
-        for (Component component : prefab.iterateComponents()) {
-            Component newComp = componentLibrary.copy(component);
-            components.add(newComp);
-            if (newComp instanceof LocationComponent) {
-                LocationComponent loc = (LocationComponent) newComp;
-                loc.setWorldPosition(position);
-            }
-        }
-        components.add(new EntityInfoComponent(prefab, prefab.isPersisted(), prefab.isAlwaysRelevant()));
-        return create(components);
+    public EntityRef create(String prefab, Vector3f position) {
+        return globalCache.create(prefab, position);
     }
 
     @Override
     public EntityRef create(Prefab prefab) {
-        List<Component> components = Lists.newArrayList();
-        for (Component component : prefab.iterateComponents()) {
-            components.add(componentLibrary.copy(component));
-        }
-        components.add(new EntityInfoComponent(prefab, prefab.isPersisted(), prefab.isAlwaysRelevant()));
-        return create(components);
+        return globalCache.create(prefab);
     }
 
     @Override
+    //Todo: Depreciated, maybe remove? Not many uses
```

Comment:
`EntityManager` says that it's depreciated in favour of `EntityRef.copy()`. Creating a builder from an existing entity is an interesting idea, especially if entities need to be duplicated multiple times (e.g. maybe for duplicating inventory items).